R/derive_extreme_event.R

```diff
@@ -199,9 +319,17 @@
 #'   )
 #' )
 #'
-#' # Derive confirmed best overall response (using event_joined())
-#' # CR - complete response, PR - partial response, SD - stable disease
-#' # NE - not evaluable, PD - progressive disease
+#' @caption A more complex example: Confirmed Best Overall Response
+#' @info The final example showcases a use of `derive_extreme_event()`
+#'   to calculate the Confirmed Best Overall Response (CBOR) in an
+#'   ADRS dataset. This example builds on all the previous ones and
+#'   thus assumes a baseline level of confidence with
+#'   `derive_extreme_event()`.
+#'
+#'   The following ADSL and ADRS datasets will be used
```

Comment:
```suggestion
#'   The following `ADSL` and `ADRS` datasets will be used
```

Reply:
General nice convention i feel - i see same in other places so will stop commenting this and feel free to update the other cases if you agree